src/js/components/modals/JobFormModal.js

```diff
@@ -54,6 +55,16 @@ const responseAttributePathToFieldIdMap = {
   '/labels': 'labels'
 };
 
+let cleanJSONspec = function (jsonDefinition) {
```

Comment:
I actually thought about something different. I thought we could utilize https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify specifically the toJSON function.